jsx/Form.js

```diff
@@ -255,6 +255,199 @@ var SelectElement = React.createClass({
   }
 });
 
+var ListElement = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string.isRequired,
+    id: React.PropTypes.string.isRequired,
+    pendingValKey: React.PropTypes.string.isRequired,
+    options: React.PropTypes.object,
+    items: React.PropTypes.array,
+    label: React.PropTypes.string,
+    value: React.PropTypes.oneOfType([
+      React.PropTypes.string,
+      React.PropTypes.array
+    ]),
+    class: React.PropTypes.string,
+    multiple: React.PropTypes.bool,
+    disabled: React.PropTypes.bool,
+    required: React.PropTypes.bool,
+    emptyOption: React.PropTypes.bool,
+    hasError: React.PropTypes.bool,
+    errorMessage: React.PropTypes.string,
```

Comment:
What's the point of having both an errorMessage and hasError? an empty error message should imply no error and vice versa.

Reply:
i agree, i was following examples of other components (Select & File)
i am guessing that hasError was necessary because errorMessage is given a non-empty default value?

as an aside i am not sure why other form components do not have props for handling error messages

Reply:
I think errorMessage is mainly to allow for a default error message behavior. I'm currently working on handling error messages of textboxes, but I think they should be available to all form components to allow for nice front-end form validation warnings.